index.html

```diff
@@ -123,69 +123,69 @@ <h3><em>Convert a Scratch project to HTML</em></h3>
         <div class="notice start-warning hidden" id="js-enabled">
           <p>
             If you see this, you can check to see if there were
-            <a href="#errors">any issues</a> loading the HTMLifier. Make sure
+            <a href="#errors" title="Errors">any issues</a> loading the HTMLifier. Make sure
             you're using a modern browser.
           </p>
           <p class="hidden" id="no-nullish-coalescing">
             Your browser does not support modern JavaScript features; check to
             make sure
             <a
               href="https://caniuse.com/mdn-javascript_operators_nullish_coalescing"
-              >your browser is modern</a
-            >.
+              title="Check if your browser is modern">your browser is modern.</a
+            >
```

Comment:
I prefer to not format punctuation marks if only a phrase (as opposed to the whole sentence) is formatted

Reply:
Examples:

```md
✔️ Please refer to the [documentation](#documentation).
❌ Please refer to the [documentation.](#documentation)
❓ Please refer to [the documentation](#documentation).
✔️ Please [refer to the documentation](#documentation).
✔️ [Please refer to the documentation.](#documentation)
```